Add postmortem feature updates (#36690)
DataDog/documentation: content/en/incident_response/incident_management/post_incident/postmortems.md

## Generate a postmortem

> {{< img src="/incident_response/incident_management/post_incident/postmortems/post_incident_tab_generate_postmortem.png" alt="The Post-Incident tab showing the Generate Postmortem button, template preview, and Follow-Ups sidebar" style="width:100%;" >}}

Reviewer: ```suggestion
{{< img src="/incident_response/incident_management/post_incident/postmortems/post_incident_tab_generate_postmortem.png" alt="The Post-Incident tab showing the Generate Postmortem button, template preview, and Follow-Ups sidebar." style="width:100%;" >}}
```

After an incident is resolved, you can generate a postmortem from the incident's **Post-Incident** tab.

To generate a postmortem:

1. Open the incident and go to the **Post-Incident** tab.
1. Select a postmortem template.
1. Click **Generate Postmortem**. Datadog creates the postmortem in the destination configured in the template and links it to the incident.

After generating a postmortem:

- **Datadog Notebooks**: The postmortem embeds directly in the **Post-Incident** tab.
- **Confluence or Google Drive**: A link appears in the **Post-Incident** tab. Click the link to open the document in the configured destination.

## View and edit a postmortem

Postmortems generated as Datadog Notebooks embed directly in the **Post-Incident** tab. You can read and edit the postmortem without leaving the incident view. Changes made in the embedded view are reflected in the underlying notebook.

> Multiple users can edit an embedded postmortem at the same time. Active editors are indicated by markers showing where each user is working. You can also add inline comments from within the embedded view.

Reviewer: ```suggestion
Multiple users can edit an embedded postmortem at the same time. Cursor markers show where each user is working. You can also add inline comments from within the embedded view.
```
